account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdateEmailHandler.java

```diff
@@ -149,6 +147,36 @@ public APIGatewayProxyResponseEvent updateEmailRequestHandler(
                                             new UserNotFoundException(
                                                     "User not found with given email"));
 
+            AtomicReference<EmailCheckResultStatus> resultStatus =
+                    new AtomicReference<>(EmailCheckResultStatus.PENDING);
+            dynamoEmailCheckResultService
+                    .getEmailCheckStore(updateInfoRequest.getReplacementEmailAddress())
+                    .ifPresent(result -> resultStatus.set(result.getStatus()));
+            LOG.info(
+                    "UpdateEmailHandler: Experian email verification status: {}",
+                    resultStatus.get());
+            if (resultStatus.get().equals(EmailCheckResultStatus.PENDING)) {
+                auditService.submitAuditEvent(
+                        AccountManagementAuditableEvent.EMAIL_FRAUD_CHECK_BYPASSED,
+                        input.getRequestContext()
+                                .getAuthorizer()
+                                .getOrDefault("clientId", AuditService.UNKNOWN)
+                                .toString(),
+                        ClientSessionIdHelper.extractSessionIdFromHeaders(input.getHeaders()),
+                        sessionId,
+                        userProfile.getSubjectID(),
+                        updateInfoRequest.getReplacementEmailAddress(),
+                        IpAddressHelper.extractIpAddress(input),
+                        userProfile.getPhoneNumber(),
+                        PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders()),
+                        AuditHelper.buildRestrictedSection(input.getHeaders()),
+                        AuditService.MetadataPair.pair(
+                                "journey_type", JourneyType.ACCOUNT_MANAGEMENT.getValue()),
+                        AuditService.MetadataPair.pair(
+                                "assessment_checked_at_timestamp", NowHelper.now()),
+                        AuditService.MetadataPair.pair("iss", AuditService.COMPONENT_ID));
```

Comment:
`iss` value does not match required value. Probably the same with `component_id`:

```
{
  "component_id": "<auth.account.gov.uk, auth.ENVIRONMENT.account.gov.uk in non-Prod>",
  ...
  "extensions": {
    ...
    "iss": "<auth.account.gov.uk, auth.ENVIRONMENT.account.gov.uk in non-Prod>"
  }
}
```